worlds/oot_soh/location_access/overworld/hyrule_field.py

```diff
@@ -5,6 +5,8 @@
     
 class EventLocations(str, Enum):
     HYRULE_FIELD = "Hyrule Field"
+    HF_CAN_DEFEAT_BIG_POE = "HF Can Defeat Big Poe"
+    HF_CAN_BORROW_BUNNY_HOOD = "HF Can Borrow Bunny Hood"
```

Comment:
Could you rename them to describe what they are? So for example something like `HF_BIG_POE` and `HF_RUNNING_MAN`